gradle/docker.gradle

```diff
@@ -149,8 +149,7 @@ subprojects {
     upgradeTest.configure(dockerConfig)
     repeatUpgradeTest.configure(dockerConfig)
 
-    // ACCEPTANCE TEST NEEDS DOCKER-COMPOSE TO WORK WITHIN DOCKER FIRST
-    // acceptanceTest.configure(dockerConfig)
-    // repeatAcceptanceTest.configure(dockerConfig)
+    acceptanceTest.configure(dockerConfig)
```

Comment:
I'm not sure acceptanceTests are currently stable with the docker configuration.  I remember something about Docker-in-docker causing a Docker-ception dock-splosion.  I'll be surprised if the `AcceptanceTest` job clears in the precheckin.

@rhoughton-pivot @smgoller @dickcav @pivotal-jbarrett, do one of you know definitively?

Reply:
> I think it's better behavior for someone running these targets to get docker if they ask for docker, rather than having the docker flags (but not the parallel forks!!!) silently ignored. 

Yeah, that's an excellent point.